packages/harmony/src/components/button/Button.module.css

```diff
@@ -0,0 +1,196 @@
+/* ===Base Styles=== */
+.button {
+  --button-color: var(--primary);
+  --text-color: var(--static-white);
+  --overlay-color: transparent;
+  --overlay-opacity: 0;
+  border: 1px solid var(--button-color);
+  border-radius: var(--unit-1);
+  color: var(--text-color);
+}
+
+/* Overlay used for hover/press styling */
+.button::before {
+  content: '';
+  position: absolute;
+  display: block;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--overlay-color, transparent);
+  opacity: var(--overlay-opacity, 0);
+  pointer-events: none;
+}
+
+.icon,
+.text {
+  z-index: 1;
+}
+
+/* === Sizes === */
+
+/* Small */
+.buttonSmall {
+  gap: var(--unit-1);
+  height: var(--unit-8);
+  padding: var(--unit-2) var(--unit-3);
+}
+
+.iconSmall {
+  width: var(--unit-4);
+  height: var(--unit-4);
+}
+
+.textSmall {
+  font-size: var(--font-s);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-4);
+  text-transform: capitalize;
+}
+
+/* Default */
+.buttonDefault {
+  gap: var(--unit-2);
+  height: var(--unit-12);
+  padding: var(--unit-3) var(--unit-6);
+}
+
+.iconDefault {
+  width: var(--unit-5);
+  height: var(--unit-5);
+}
+
+.textDefault {
+  font-size: var(--font-l);
+  font-weight: var(--font-bold);
+  line-height: calc(4.5 * var(--unit));
+  text-transform: capitalize;
+}
+
+/* Large */
+.buttonLarge {
+  gap: var(--unit-2);
+  height: var(--unit-15);
+  padding: var(--unit-5) var(--unit-6);
+}
+
+.iconLarge {
+  width: var(--unit-6);
+  height: var(--unit-6);
+}
+
+.textLarge {
+  font-size: var(--font-xl);
+  font-weight: var(--font-bold);
+  line-height: var(--unit-5);
+  letter-spacing: 0.25px;
+  text-transform: uppercase;
+}
+
+/* === Color Variants === */
+
+/* Primary */
+.primary {
+  --text-color: var(--static-white);
+  --button-color: var(--primary);
+  background: var(--button-color);
+  box-shadow: var(--shadow-near);
+}
+
+.primary:hover {
+  --overlay-color: var(--static-white);
+  --overlay-opacity: 0.1;
+  box-shadow: var(--shadow-mid);
+}
+
+.primary:active {
+  --overlay-color: var(--static-black);
```

Comment:
not sure if static black exists? can probably just use black for this seeing as its just for an overlay

Reply:
i added static-black. Seems like the extremes would be useful, but we can remove it too since it wasn't in the spec